### PR TITLE
Update rubygem-foreman-tasks to 0.17.6

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

**Comment by adamruzicka** [comment]
Oh boy, `react-html-parser` is not packaged yet, this will be fun.

**Comment by adamruzicka** [comment]
@ekohl Do I read it correctly I'll need to bring that `react-html-parser` in in a separate PR?

**Comment by ekohl** [comment]
Correct. It's a build dependency (it's listed as a `BuildRequire`) and our CI job can't use results built in the same PR.

**Comment by zjhuntin** [comment]
@adamruzicka did the PR to add `react-html-parser` get created?

**Comment by adamruzicka** [comment]
Only for develop so far (https://github.com/theforeman/foreman-packaging/pull/4768 and https://github.com/theforeman/foreman-packaging/pull/4835)

**Comment by pcreech** [comment]
[test rpm]

**Comment by pcreech** [comment]
[test rpm]

File: package_manifest.yaml [hunk 1]
```diff
@@ -368,6 +368,7 @@ foreman_nonscl_packages:
     katello-certs-tools: {}
     katello-selinux: {}
     keycloak-httpd-client-install: {}
+    nodejs-react-html-parser: {}
```
Comment (pcreech) [inline package_manifest.yaml:371]: I think this needs to be moved up to the foreman_nodejs_packages group above this group.  It's whitelisted for scl, but the foreman_nonscl_packages group doesn't build against that tag.

Comment (pcreech) [reply package_manifest.yaml:371]: Also, I think this needs to be broken out into it's own PR, as it seems this is a new build time dependency for foreman-tasks

**Comment by ehelms** [comment]
@adamruzicka Given we are starting the branch process for Foreman 2.2, can I close this?

**Comment by adamruzicka** [comment]
Alright, let's scratch this

**Pull request closed without merging**